examples/reminderbot/README.md

```diff
@@ -24,14 +24,37 @@ To train and chat with `reminderbot`, execute the following steps:
     rasa train
     ```
     The model will be stored in the `/models` directory as a zipped file.
-    
-2. Run a Rasa action server with
+
+2. Run a Rasa SDK action server with
     ```
     rasa run actions
     ```
+
+3. (Option 1) Run Rasa X to talk to your bot. In a separate console window from where you ran the step 2 command:
+    ```
+    rasa x
+    ```
+
+3. (Option 2) To test this example without Rasa X, run a 
+   [callback channel](https://rasa.com/docs/rasa/user-guide/connectors/your-own-website/#callbackinput). 
+   In a separate console window from where you ran the step 2 command:
+    ```
+    python callback_server.py
+    ```
+     
+   This will run a server that prints the bot's responses to the console. 
+   
+   Start your Rasa server in a third console window: 
+   ```
+   rasa run --enable-api 
+   ```
    
-3. Run a Rasa X to talk to your bot. 
-   If you don't have a Rasa X server running, you can test things with `rasa x` in a separate shell (the action server must keep running).
+   You can then send messages to the bot via the callback channel endpoint: 
+   ```
+   curl -XPOST http://localhost:5005/webhooks/callback/webhook \
+      -d '{"sender": "tester", "message": "hello"}' \
+      -H "Content-type: application/json"
+   ```
 

```

Comment:
Yes this worked great, thank you for writing the instructions & callback server!

I defined a function to make testing easier (I have the same for duckling et al) but I think this is shell specific so should not go in the instructions?

```
callback() {
d="{\"sender\": \"tester\", \"message\": \"$@\"}"
echo $d 
curl -XPOST http://localhost:5005/webhooks/callback/webhook \
      -d $d \
      -H "Content-type: application/json"
}
```

Reply:
Ooh nice! I usually just test by sending messages via insomnia 😅